diff --git a/ihmc_ros_java_adapter/build.gradle b/ihmc_ros_java_adapter/build.gradle
index 5311d46..0686105 100644
--- a/ihmc_ros_java_adapter/build.gradle
+++ b/ihmc_ros_java_adapter/build.gradle
@@ -34,7 +34,11 @@ repositories {
     maven {
         url 'http://updates.jmonkeyengine.org/maven'
     }
-
+    
+    maven{
+	url 'https://artifactory.ihmc.us/artifactory/releases'
+    }
+    
     mavenLocal()
 
     jcenter()
@@ -117,6 +121,10 @@ task runJavaDelegate(type: JavaExec, dependsOn: parseYaml) {
     if(!project.hasProperty("yaml")) {
         return
     }
+    configurations {
+      all*.exclude group: 'org.bytedeco', module: 'javacpp-presets'
+    }
+    
     ignoreExitValue = true
     classpath = configurations.ros
     workingDir = project.projectDir
