diff --git a/ihmc_ros_common/launch/robot_description_common.launch b/ihmc_ros_common/launch/robot_description_common.launch
index d619cc3..420ef6c 100644
--- a/ihmc_ros_common/launch/robot_description_common.launch
+++ b/ihmc_ros_common/launch/robot_description_common.launch
@@ -6,8 +6,13 @@
   
   <param name="/ihmc_ros/robot_name" value="$(arg robot_name)" />
 
-  <node name="robot_state_publisher" pkg="robot_state_publisher" type="state_publisher" >
-    <remap from="joint_states" to="/ihmc_ros/$(arg robot_name)/output/joint_states" />
+  <node name="joint_state_publisher" pkg="joint_state_publisher" type="joint_state_publisher">
+     <param name="use_gui" value="FALSE"/>     
+     <param name="rate" value="500"/>     
+     <rosparam param="/source_list">[/multisense/joint_states, /hardware_joint_states]</rosparam>      
+  </node>
+  <node name="robot_state_publisher" pkg="robot_state_publisher" type="robot_state_publisher" >
+    <!-- remap from="joint_states" to="/ihmc_ros/$(arg robot_name)/output/joint_states" / -->
     <remap from="robot_description" to="/$(arg robot_name)/robot_description" />
   </node>
 </launch>
diff --git a/ihmc_ros_java_adapter/build.gradle b/ihmc_ros_java_adapter/build.gradle
index 5311d46..fd4a586 100644
--- a/ihmc_ros_java_adapter/build.gradle
+++ b/ihmc_ros_java_adapter/build.gradle
@@ -35,6 +35,9 @@ repositories {
         url 'http://updates.jmonkeyengine.org/maven'
     }
 
+    maven{
+	url 'https://artifactory.ihmc.us/artifactory/releases'
+    }
     mavenLocal()
 
     jcenter()
@@ -117,6 +120,9 @@ task runJavaDelegate(type: JavaExec, dependsOn: parseYaml) {
     if(!project.hasProperty("yaml")) {
         return
     }
+    configurations {
+      all*.exclude group: 'org.bytedeco', module: 'javacpp-presets'
+    }
     ignoreExitValue = true
     classpath = configurations.ros
     workingDir = project.projectDir
